CMakeLists.txt

```
@@ -609,13 +620,25 @@ endif()
 option(CURL_ZSTD "Set to ON to enable building curl with zstd support." OFF)
 set(HAVE_ZSTD OFF)
 if(CURL_ZSTD)
-  find_package(Zstd REQUIRED)
-  if(Zstd_FOUND AND NOT Zstd_VERSION VERSION_LESS "1.0.0")
-    set(HAVE_ZSTD ON)
-    list(APPEND CURL_LIBS ${Zstd_LIBRARIES})
-    include_directories(${Zstd_INCLUDE_DIRS})
+  hunter_add_package(zstd)
+  if(HUNTER_ENABLED)
+    find_package(zstd REQUIRED)
```

Is it intentional that you changed Zstd to zstd Herr? But the variable later is still Zstd_FOUND?

---

Furthermore I suggest to use `CONFIG` keyword to prevent picking up a `FindZstd.cmake` module file intervening with finding the Hunter `zstd` config

```suggestion
    find_package(zstd CONFIG REQUIRED)
```